src/main/java/com/mongodb/jdbc/MongoDatabaseMetaData.java

```diff
@@ -1539,10 +1559,12 @@ private Stream<BsonDocument> getColumnsFromDB(
                         tableName ->
                                 new Pair<>(
                                         new Pair<>(dbName, tableName),
-                                        db.runCommand(
-                                                new BsonDocument(
-                                                        "sqlGetSchema", new BsonString(tableName)),
-                                                MongoJsonSchemaResult.class)))
+                                        db.withCodecRegistry(MongoDriver.registry)
```

Comment:
Was this not working when applied at the beginning of the function?
I don't know the impact of setting the codec registry, but it seems that if we can avoid doing it for each collection, that'd be good. Worst case, it won't make a difference, best case it does not slow the stream processing.

Reply:
The runCommand() was sending an empty document with the codec registry set, I was able to resolve the issue by adding `MongoClientSettings.getDefaultCodecRegistry()` to our codec registry.